### PR TITLE
Fix category name for language worker channel logs

### LINKED ISSUE #4611: Investigate missing system logs
Logs are missing for any traces that use 

workerChannelLogger : https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs#L75  and 

workerProcessLogger: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Rpc/LanguageWorkerProcess.cs#L49

### PR DESCRIPTION
**Pull request by pragnagopa**
Fixes #4611 

cc @maiqbal11

### CODE REVIEW

**Comment by brettsam** [comment]
That list is here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Extensions/ScriptLoggingBuilderExtensions.cs. Did this category change during the refactor?

File: src/WebJobs.Script/Rpc/LanguageWorkerChannelFactory.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ public ILanguageWorkerChannel CreateLanguageWorkerChannel(string scriptRootPath,
                 throw new InvalidOperationException($"WorkerCofig for runtime: {runtime} not found");
             }
             string workerId = Guid.NewGuid().ToString();
-            ILogger workerLogger = _loggerFactory.CreateLogger($"LanguageWorkerChannel.{runtime}.{workerId}");
+            ILogger workerLogger = _loggerFactory.CreateLogger($"Worker.LanguageWorkerChannel.{runtime}.{workerId}");
```
Comment (brettsam) [inline src/WebJobs.Script/Rpc/LanguageWorkerChannelFactory.cs]: Ideally we'd have a method to construct the category -- like we have in SDK here: https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogCategories.cs#L55.

But this works for now...

**Comment by pragnagopa** [comment]
Closing this PR and will send a PR against master as the current release needs to be patched.

**Comment by pragnagopa** [comment]
> That list is here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Extensions/ScriptLoggingBuilderExtensions.cs. Did this category change during the refactor?

Yes. I changed the name of the category as part of refactoring.

**Pull request closed without merging**